setup.py

```diff
@@ -116,4 +116,10 @@ def _global_version(version):
     ],
     package_data={"vyper.ast": ["grammar.lark"]},
     data_files=[("", [hash_file_rel_path])],
+    # Add linter configuration for flake8
+    options={
+        "flake8": {
+            "select": "F401",
```

Comment:
i think we rather want this in `setup.cfg`

but also, i think F401 was already enabled, so i'm confused why it didn't catch the dead variable in question.